menubot/index.js

```diff
@@ -0,0 +1,127 @@
+import {BossRoomBot, CharInfo} from "../bossroom";
+import {RGValidator, RGBot} from "../rg";
+
+let charType = Math.round(Math.random() * 1000000) % 4;
+
+export function configureBot(characterType) {
+  console.log(`Unity bot configureBot function called, charType: ${charType} - characterType: ${characterType}`);
+  charType = CharInfo.type.indexOf(characterType);
+}
+
+let rgValidator = new RGValidator();
+
+let botComplete = false;
+
+// flags for clicking the 6 buttons we need to click to start the game
+let stateFlags = [false,false,false,false,false,false]
```

Comment:
Similar to what Will mentioned, maybe we can make this a map with the name being the purpose of the button?

Reply:
resolved